src/components/button/Button.jsx

```diff
@@ -13,9 +14,9 @@ export const Button = ({
   className
 }) => {
   const buttonClassNames = classNames(
-    'button',
+    'c-btn',
     {
-      'button--secondary': secondary,
+      [`c-btn--${variant}`]: variant,
       'cursor-not-allowed': disabled
```

Comment:
This can be removed now that you've added it in sassy

Reply:
good spot man